scipy/stats/tests/test_continuous_basic.py

```diff
@@ -87,7 +87,7 @@
                'logistic', 'loguniform', 'maxwell', 'nakagami',
                'ncf', 'nct', 'ncx2', 'norminvgauss', 'pearson3',
                'powerlaw', 'rdist', 'reciprocal', 'rice',
-               'skewnorm', 't', 'truncweibull_min',
+               'skewnorm', 't', 'truncpareto', 'truncweibull_min',
```

Comment:
Many distributions don't accept complex dtypes as input. `truncpareto` is now one of them.